spec/models/signon_identity_spec.rb

```diff
@@ -159,11 +158,15 @@
       end
 
       context 'when an admin' do
-        let!(:orgs) { [EstateSSOMapper::DIGITAL_ORG] }
+        let(:roles) { ["ROLE_MOIC_ADMIN"] }
```

Comment:
I wonder if we should have a new role made specifically for PVB admin, rather than using `ROLE_MOIC_ADMIN`.

That would make the service more portable if it ever moves to a different team.